src/sentry/issues/grouptype.py

```diff
@@ -211,6 +212,8 @@ class ProfileJSONDecodeType(GroupType):
     [
         ProfileBlockedThreadGroupType.type_id,
         ProfileFileIOGroupType.type_id,
+        ProfileImageDecodeGroupType.type_id,
```

Comment:
Not sure why this frozenset exists, you can use `get_group_types_by_category`. Will `ProfileBlockedThreadGroupType` be removed later on? I think it's just used in tests.

Reply:
Yeah, not totally sure why we have this. Maybe as a hard-coded whitelist for ingesting issue occurrence types? I'd rather not mess with it since I don't know the context.